### DISCUSSION TITLE
How to remove default metadata?

### DISCUSSION THREAD

**Discussion by statusunknown418** [trpc/trpc#4554] [Q&A]
I'm developing a multiple client app, where the backend is obviously Nextjs + TRPC, however one of the clients is a native iOS SwiftUI app and that's where I'm having the most trouble trying to setup.

All my routers are giving me an extra field called `meta` this even if I don't set it up or provide a default meta value of `undefined`.

Response looks like this:
```JSON
{
  "result": {
    "data": {
      "json": [
        {
          "id": 1,
          "name": "test",
          "description": "asometj",
          "price": 0,
          "image": null,
          "category": "MEAL",
          "starSummary": 0,
          "userId": null,
          "createdAt": "2023-06-23T00:33:41.987Z",
          "updatedAt": "2023-06-23T00:32:45.919Z"
        }
      ],
      "meta": {
        "values": {
          "0.createdAt": [
            "Date"
          ],
          "0.updatedAt": [
            "Date"
          ]
        }
      }
    }
  }
}
```

SwiftUI is giving me trouble because of the `"0.createdAt"` field, it cannot serialize an object that starts with a number, I wanted to know how to completely remove this `meta` field

**Comment by KATT** [comment]
This comes from superjson, not trpc
